src/modules/checkIsValidUserInput.js

```diff
@@ -9,7 +9,27 @@
  * @return {boolean} - True if the user input is valid, false otherwise
  */
 function checkIsValidUserInput(userInput) {
-  /* Write your code here */
+  if (userInput.length !== 4) {
+    return false;
+  }
+
+  if (isNaN(userInput)) {
+    return false;
+  }
+
+  for (let i = 0; i < userInput.length; i++) {
+    const check = userInput.slice(1);
+
+    if (userInput[0] === '0') {
```

Comment:
Checking only the first character for '0' is not sufficient, as any of the digits in the input should not be '0'. This check should be inside the loop to validate each character of `userInput`.